bot/constants.py

```diff
@@ -1,10 +1,26 @@
 import pathlib
 import os
+import typing
 
 # env vars
 PREFIX = os.getenv("PREFIX") or "!"
 TOKEN = os.getenv("TOKEN")
 
+
+class Roles(typing.NamedTuple):
+    announcements = int(os.getenv("ANNOUNCEMENTS_ID", "789978290844598272"))
+    polls = int(os.getenv("POLLS_ID", "790043110360350740"))
+
+
+class Emojis(typing.NamedTuple):
+    confirmation = "<:yes:794231332770938901>"
+    warning = "\u26a0"
```

Comment:
I believe the the emoji should be specified in the format `name_emoji`.
[Smart Config](https://github.com/python-discord/smartconfig) will supposedly detect emojis based on its suffix.
```suggestion
    confirmation_emoji = "<:yes:794231332770938901>"
    warning_emoji = "\u26a0"
```

Reply:
But it is already in the Emojis class so i don't see the need of adding `_emoji` to it.

Reply:
I don't know about Smart Config a lot but will see